evm_loader/cli/src/main.rs

```diff
@@ -182,35 +685,143 @@ fn main() {
+                        .validator(is_valid_h160)
+                        .help("Ethereum address"),
+                )
+                .arg(
+                    Arg::with_name("lamports")
+                        .long("lamports")
+                        .value_name("lamports")
+                        .takes_value(true)
+                        .default_value("0")
+                        .required(false)
+                )
+                .arg(
+                    Arg::with_name("space")
+                        .long("space")
+                        .value_name("space")
+                        .takes_value(true)
+                        .required(false)
+                        .default_value("0")
+                        .help("Length of data for new account"),
+                )
+            )
+        .subcommand(
+            SubCommand::with_name("create-program-address")
+                .about("Generate a program address")
+                .arg(
+                    Arg::with_name("seed")
+                        .index(1)
+                        .value_name("SEED_STRING")
+                        .takes_value(true)
+                        .required(true)
+                        .help("The seeds (a string containing seeds in hex form, separated by spaces)"),
+                )
+        )
+        .subcommand(
+            SubCommand::with_name("deploy")
+                .about("Deploy a program")
+                .arg(
+                    Arg::with_name("program_location")
+                        .index(1)
+                        .value_name("PROGRAM_FILEPATH")
+                        .takes_value(true)
+                        .required(true)
+                        .help("/path/to/program.o"),
+                )
+        )
         .get_matches();
 
-        let (sub_command, sub_matches) = app_matches.subcommand();
-        let matches = sub_matches.unwrap();
+        let mut wallet_manager = None;
+        let config = {
+            let cli_config = if let Some(config_file) = app_matches.value_of("config_file") {
+                solana_cli_config::Config::load(config_file).unwrap_or_default()
+            } else {
+                solana_cli_config::Config::default()
+            };
 
-        let cli_config = if let Some(config_file) = matches.value_of("config_file") {
-            solana_cli_config::Config::load(config_file).unwrap_or_default()
-        } else {
-            solana_cli_config::Config::default()
+            let json_rpc_url = normalize_to_url_if_moniker(
+                app_matches
+                    .value_of("json_rpc_url")
+                    .unwrap_or(&cli_config.json_rpc_url),
+            );
+
+            let evm_loader = pubkey_of(&app_matches, "evm_loader")
+                    .unwrap_or_else(|| {
+                        eprintln!("Need specify evm_loader");
+                        exit(1);
+                    });
+
+            let verbose = app_matches.is_present("verbose");
```

Comment:
verbose flag does nothing. May be remove it? Or use it and print detailed logs.

Reply:
I added `log` module and used `verbose` flag to sets its level.

Reply:
CLI imports the code from the evm_loader. How `log` module will interact with `debug_print!` from there?